src/frontend/src/lib/components/hero/HeroContent.svelte

```diff
@@ -24,7 +24,10 @@
 </script>
 
 {#if summary}
-	<div transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}>
+	<div
+		transition:slide={{ delay: 0, duration: 250, easing: quintOut, axis: 'y' }}
+		class="flex flex-col items-center"
+	>
 		<div class="icon flex justify-between items-center mt-6 md:mt-12 mb-0.5 pt-2">
 			<div>
```

Comment:
Do we still need that div?

Reply:
not really, i didn't remove it because it was already there and I was not sure why. Shall I remove? I tested and it doesn't change anything

Reply:
Yes. It's on main but, after your recent changes it is not necessary anymore as far as I can tell.

Reply:
I can try too

Reply:
i will remove it

Reply:
as discussed, I updated it